libs/model/src/user/GetUserProfile.query.ts

```diff
@@ -8,7 +8,7 @@ export function GetUserProfile(): Query<typeof schemas.GetUserProfile> {
   return {
     ...schemas.GetUserProfile,
     auth: [],
-    secure: true,
+    secure: false,
     body: async ({ actor, payload }) => {
       const user_id = payload.userId ?? actor.user.id;
```

Comment:
add actor.user?.id in line 13 just in case payload is empty

Reply:
should also check that user is valid after loading it, can use mustExist guard

Reply:
done thanks